src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php

```diff
@@ -54,6 +54,10 @@ public function logKernelException(GetResponseForExceptionEvent $event)
         $this->logException($event->getException(), sprintf('Uncaught PHP Exception %s: "%s" at %s line %s', $e->getClass(), $e->getMessage(), $e->getFile(), $e->getLine()));
     }
 
+    /**
+     * @param string                   $eventName
+     * @param EventDispatcherInterface $eventDispatcher
+     */
```

Comment:
Are you sure?

Reply:
yes, maps to L76 in the file: https://github.com/symfony/symfony/pull/30691/files#diff-3a012244a1c6d5c901c4f26d6b2ca299R76

Reply:
But the first argument is not a string, it it?

Reply:
I don't understand this either.

Reply:
We do partial docblocks: $event is already documented in the signature - this document the two other arguments that are passed to the method when it is being called.
This is required to prepare making them explicit in 5.0, and remove the `func_get_arg(2)` L76.